cli/npm/resolvers/local.rs

```diff
@@ -531,3 +569,42 @@ fn join_package_name(path: &Path, package_name: &str) -> PathBuf {
   }
   path
 }
+
+#[cfg(test)]
+mod test {
+  use deno_npm::NpmPackageCacheFolderId;
+  use deno_semver::npm::NpmPackageNv;
+
+  use super::*;
+
+  #[test]
+  fn test_get_package_folder_id_folder_name() {
```

Comment:
I was about to comment that it would be good to unit test `get_package_folder_id_from_folder_name` but I see you got it covered :)